cheatsheets_excluded/Content_Security_Policy_Cheat_Sheet.md

```diff
@@ -1,203 +1,200 @@
----
-title: Content Security Policy Cheat Sheet
-permalink: /Content_Security_Policy_Cheat_Sheet/
----
+# Introduction
```

Comment:
To finalise the refactoring, can you move the CSP CS MD file from the folder cheatsheets_excluded to the folder cheatsheets ?
Thanks you very much in advance 😃